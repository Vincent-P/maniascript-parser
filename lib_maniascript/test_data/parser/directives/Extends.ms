#Extends "Modes/ShootMania/Base/ModeShootMania.Script.txt"
