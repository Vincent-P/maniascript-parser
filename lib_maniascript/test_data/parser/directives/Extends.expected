NODE_ROOT 0..58 {
  NODE_EXTENDS 0..58 {
    TOKEN_EXTENDS("#Extends") 0..8
    TOKEN_WHITESPACE(" ") 8..9
    NODE_STRING 9..58 {
      TOKEN_STRING_START("\"") 9..10
      TOKEN_STRING_CONTENT("Modes/ShootMania/Base/ModeShootMania.Script.txt") 10..57
      TOKEN_STRING_END("\"") 57..58
    }
  }
}